kustomize/overlays/sandbox/kustomization.yaml

```diff
@@ -17,13 +17,11 @@ bases:
 # Contour is used to create ingress. On cloud service use the default provided ingress controllers or cloud LB's
 # Add node ports for ease of use locally
 resources:
-  - ../../base/operators/spark
-  - ../../base/operators/kfoperators/pytorch
   - ./dependencies/database
   - ./dependencies/storage
-  - ./dependencies/redis
   - ./dependencies/contour_ingress_controller
```

Comment:
Should we lose `contour` in favor of bare `envoy`?

Reply:
Ah, alas, this may not be worth it since `ingress` is defined in the `single_cluster/headless` base anyway. Ultimately, it would make sense to make `ingress` optional and only define a core set of modules in `single_cluster/headless`.